### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by kensteph**
Hi there 👋!
In this PR I : 
✅ Add tests for the Game class and all implemented methods.
✅ Add tests for the Author class and all implemented methods.

### CODE REVIEW

File: spec/game_spec.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      expect(published_date).to eql '2020-05-14'
+    end
+  end
+
+  describe '#can_be_archived?' do
+    it 'Should return true' do
+      can_be_archived = @game.can_be_archived?
+
+      expect(can_be_archived).to be false
+    end
```
Comment (biruk-bereka) [inline spec/game_spec.rb]: Please kindly change `can_be_archived?` method to a private method in the game class as described in the UML diagram. Here you call as public method. And update the test for this method. 

In RSpec, private methods are not intended to be directly tested as they are implementation details and should not be exposed through the public interface of the class. However, if you need to test a private method for some reason, there are a few approaches you can take.

One option is to use the send method to invoke the private method within your test. For example, suppose you have a class MyClass with a private method my_private_method, you can test it like this:

            describe MyClass do
                 describe "#my_private_method" do
                       it "returns the expected result" do
                                 instance = MyClass.new
                                 expect(instance.send(:my_private_method, arg1, arg2)).to eq(expected_result)
                       end
                end
           end

**Review by biruk-bereka — CHANGES_REQUESTED** [review]
Hi @kensteph 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some improvements, but you are almost there!

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: spec/game_spec.rb (hunk 1 above)
Comment (kensteph) [reply spec/game_spec.rb]: Thank you @biruk-bereka  for your review.

**Review by biruk-bereka — APPROVED** [review]
Hi @kensteph, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design
✔️  All test case passed

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**